src/pages/index.tsx

```diff
@@ -80,6 +80,21 @@ export const getStaticProps: GetStaticProps<{
       : content.resources.push(item)
   );
 
+  // Temp solution for lecture sort order based on requirements doc - [DEVHUB-1439]
+  const lectureSortOrder = [
+    'MongoDB 101 : Non-Relational Databases for Beginners',
```

Comment:
it looks like the `id` for each item in Strapi is the same for staging vs. prod, I can list out id's here instead if we're concerned about someone editing the `title`

Reply:
Hmm yeah maybe do it on the id. Hopefully this isn't a long term solution but should provide some stability at least.